### ISSUE TITLE
Croatian and Serbian translations

### ISSUE THREAD

**Issue by Changaco** [liberapay/salon#593]
(Moving the request made by @Susexe in https://github.com/liberapay/salon/issues/496#issuecomment-1829644404 to a separate issue.)

> Could you also include the Core component for Serbian and Croatian? The languages are very similar, so once the Serbian translation is done, I can run a Cyrillic-to-Latin transliteration script and review strings by hand.

Done:

- Serbian: https://hosted.weblate.org/projects/liberapay/core/sr/
- Croatian: https://hosted.weblate.org/projects/liberapay/core/hr/

**Comment by b0c0de** [comment]
@Changaco 

Two things: 

1. Serbian and Croatian are similar but some terms and structure of sentence is different. What @Susexe suggested would not work because it would not be proper translation. I can, however, translate to both languages if you would like. 
2. When I tried starting to contribute to translate to Serbian, it gives me following error 

**Insufficient privileges for saving translations.**

What do I need to do in order to start translating ? 

Please, advise 

Cheers!

**Comment by jorgesumle** [comment]
> Insufficient privileges for saving translations.
> What do I need to do in order to start translating ?

You need a Weblate account or, alternatively, you can sign in with a third-party app. I could edit the Serbian translation without any issue. How did you sign in?

**Comment by b0c0de** [comment]
@jorgesumle 

I was not aware we needed to sign in. I will try later tonight and see if it works as I am busy right now with other projects I am working on. 

Thanks !
